src/R/Editor/Impl/Selection/SelectWordCommand.cs

```diff
@@ -24,60 +22,13 @@ public sealed class SelectWordCommand : ViewCommand {
         public override CommandResult Invoke(Guid group, int id, object inputArg, ref object outputArg) {
```

Comment:
Ah, okay, so I did understand it right. I think they should be doing that (or at least it's worth trying to convince them that they should).